src/styles/main.scss

```diff
@@ -1,3 +1,65 @@
+$calendar-width: 707px;
+
+* {
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  width: $calendar-width;
+  margin: 100px auto;
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 1px;
```

Comment:
You're gonna need a variable for this as well in order to calculate everything correctly